README.md

```diff
@@ -1,22 +1,29 @@
 # BreweryDB API Wrapper in Golang
 
+[![Build Status](https://travis-ci.org/hobbeswalsh/brewerydb.svg?branch=master)](https://travis-ci.org/hobbeswalsh/brewerydb)
+
 ## To Use:
 
-```
+```go
+    package main
+
     import (
     	"github.com/michaelporter/brewerydb"
     )
+    
+    c := brewerydb.NewClient("myApiKey")
+
+    // I want a random beer.
+    beer, err := c.RandomBeer()
```

Comment:
Looks like RandomBeer() requires a BeerQuery param; can you update the README to illustrate that?